exporters/otlp/common/src/test/java/io/opentelemetry/exporter/internal/auth/AuthenticatingExporterTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.internal.auth;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
```

Comment:
We use assertj on this project. Can you replace these with the assertj equivalent? I.e. `assertThat(...).isNull()`